.travis.yml

```diff
@@ -11,12 +11,15 @@ env:
   - PIP_DIR="$HOME/virtualenv"
   - GATK_PATH="$CACHE_DIR/GenomeAnalysisTK-3.6"
   - PYTHONIOENCODING=UTF8
-  #- DOCKER_REGISTRY="docker.io"
-  #- DOCKER_REPO_PROD="broadinstitute/viral-ngs"
-  #- DOCKER_REPO_DEV="broadinstitute/viral-ngs-dev"
+  
   - DOCKER_REGISTRY="quay.io"
   - DOCKER_REPO_PROD="quay.io/broadinstitute/viral-ngs"
   - DOCKER_REPO_DEV="quay.io/broadinstitute/viral-ngs-dev"
+  
+  - DOCKER_REGISTRY_MIRROR="docker.io"
+  - DOCKER_REPO_PROD_MIRROR="broadinstitute/viral-ngs"
+  #- DOCKER_REPO_DEV_MIRROR="broadinstitute/viral-ngs-dev"
```

Comment:
`docker.io/broadinstitute/viral-ngs-dev` no longer exists, and it looks like your shell script modifications below don't do anything with $DOCKER_REPO_DEV_MIRROR anyway?